views/user-welcome.twig

```diff
@@ -17,6 +17,55 @@
     .tour .popover-title{
         background-color: #b2bfdc;
     }
+
+    /* Print Style Sheet - for POP reports */
+    @media print {
+
+        @page {
+            margin: 0;
+            size: landscape;
+        }
+
+        body  {
+            margin: 1.6cm;
+            font-family: "Helvetica Neue",Helvetica,Arial,sans-serif;
+            font-size: 14px;
+        }
+
+        .nav-pills, .proof-of-play-report-div, #applyBtn, #applyWarning, .fa-info-circle , .dt-buttons, .pagination,
+        .select2-selection__clear, button.select-all, button.dropdown-toggle
+        {
+            display: none;
+        }
+
+        .widget .widget-title {
+            font-size: 20px;
+            color: #767676;
+            background-color: #f6f6f6;
+            text-align: center;
+            line-height: 60px;
+        }
+
+        .well {
+            border: 2px solid #e3e3e3;
+        }
+
+        /* Reports of summary, distribution */
+        #reportChart {
+            max-width: 25cm;
+            max-height: 7cm;
+        }
+
+        /* Custom reports*/
+        #campaignChart, #sensor1Chart, #sensor2Chart, #sensor3Chart {
```

Comment:
We're going to need to handle this differently as we can't name all of the reports individually.

Can you set some generic class names for this function, and then apply those to the reports?